scripts/jenkins/qa/cloud-mkphyscloud-qa-test-horizon-ui-integration.yaml

```diff
@@ -0,0 +1,71 @@
+- job:
+    name: cloud-mkphyscloud-qa-test-horizon-ui-integration
```

Comment:
As horizon is the UI for openstack, perhaps, the `-ui-` is a bit redundant in the name; something like `horizon_tests` should be ok, and putting more details in the description would provide more information on it.